testnet/bitcoin-neon-controller/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM rust as builder
+WORKDIR /src/bitcoin-neon-controller
+COPY . .
+RUN cd /src/bitcoin-neon-controller && cargo build --target x86_64-unknown-linux-gnu --release
+RUN cargo install --target x86_64-unknown-linux-gnu --path /src/bitcoin-neon-controller
+
+FROM alpine
+ARG GLIBC_VERSION="2.31-r0"
```

Comment:
Why is the Docker file installing glibc?  The bitcoin neon controller works just fine with musl -- I've been using it for testing for the last week without issues.

Reply:
will defer to @wileyj, who wrote the Dockerfile